assets/css/styles.css

```diff
@@ -89,3 +91,158 @@ ul {
 .social {
   width: 15px;
 }
+
+.recent-works-title {
+  background: url("../../assets/img/Indicator.png") 0 0 no-repeat;
+  background-position: bottom center;
+}
+
+.scroll-down {
+  background-color: rgba(0, 0, 0, 0);
+  border-style: hidden;
+  margin: 3em 0 0;
+}
+
+.disabled-button:hover {
+  background-color: #1e488f;
+}
+
+.works-wrapper {
+  display: grid;
+  grid-template-rows: 474px 474px 474px 474px 474px 474px;
+  gap: 2em;
+  margin: 3em 0 1em 0;
+}
+
+.works-card {
+  background-size: 100%;
+  flex-direction: column;
+  justify-content: flex-end;
+  border-radius: 8px;
+  border: 1px solid #d0d9d7;
+}
+
+.project1 {
+  background: url("../../assets/img/1.png") 0 0 no-repeat;
+}
+
+.project2 {
+  background: url("../../assets/img/2.png") 0 0 no-repeat;
+}
+
+.project3 {
+  background: url("../../assets/img/3.png") 0 0 no-repeat;
+}
+
+.project4 {
+  background: url("../../assets/img/4.png") 0 0 no-repeat;
+}
+
+.project5 {
+  background: url("../../assets/img/5.png") 0 0 no-repeat;
+}
+
+.project6 {
+  background: url("../../assets/img/6.png") 0 0 no-repeat;
+}
+
+.skill-wrap {
+  flex-wrap: wrap;
+  justify-content: space-around;
+  padding: 0;
+  width: 100%;
+}
+
+.works-title {
+  font-weight: 600;
+  font-size: 24px;
+  line-height: 32px;
+  color: #3a4a42;
+  margin: 0;
+}
+
+.work-skill {
+  font-weight: 600;
+  font-size: 12px;
+  line-height: 16px;
+  color: #3a4a42;
+  background-color: #ebf0ee;
+  border-radius: 4px;
+  padding: 8px 12px;
+}
+
+.works-card-inner {
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  border-radius: 8px;
+  padding: 16px 0;
+  width: 100%;
+  align-items: center;
+}
+
+.home-btn {
+  border-radius: 4px;
+  width: 40%;
+  background: #36b37f;
+  height: 48px;
+  color: #fff;
+  border: 0;
+  margin: 1em 0;
+}
+
+.home-btn:hover {
+  background: #1e488f;
+  color: #fff;
+}
+
+.about-wrapper {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 2em;
+}
+
+.about-header {
+  margin-bottom: 1em;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background: url("../../assets/img/about-me-1.png") 0 100% no-repeat, url("../../assets/img/Mask-Group.png") 100% 0 no-repeat;
+}
```

Comment:
- [ ] **[OPTIONAL]** Please, 🙏🏽  add a margin for the paragraph to look  similar to the template:

> You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

- Figma template

![image](https://user-images.githubusercontent.com/58642949/185727551-341f053d-c015-4a22-ad68-de4f3a159c13.png)

- Your design

![image](https://user-images.githubusercontent.com/58642949/185727558-696ad0e4-9277-4129-a89e-34353dc05024.png)

Reply:
Thanks for this comment 👍 .
The margin has been added as advised.